### ISSUE TITLE
Create an Homebrew installer

### ISSUE THREAD

**Issue by g4challenge** [alyssais/secure-pbcopy#1]
Thanks for the project, it does what it should to do.

Would it be possible to make it an `homebrew tap`?

as on macos this can help with cmd-line password managers.

**Comment by alyssais** [comment]
Yes! I'd be happy for there to be a Homebrew package. I don't use Homebrew any more (I switched to Nix), so I probably shouldn't be possible for maintaining it. Ideally it would be in homebrew/core but they frown on packages that don't meet notability requirements that this project isn't really close to meeting (30 forks, 30 watchers, or 75 stars).

Since that isn't very likely for how, the best thing would be if somebody who uses Homebrew would manage the tap. If somebody wants to volunteer to do this I'll mention it in the README. If you would like to, let me know. I'd be happy to help with the packaging – I used to be a Homebrew maintainer.

**Comment by KiligWYu** [comment]
Thanks for the project, it solved my problem. Thanks!
